sematic/log_reader.py

```diff
@@ -48,8 +47,27 @@ def get_resolution(self, resolution_id: str) -> Resolution:
     def get_run(self, run_id: str) -> Run:
         return self.value[0].get_run(run_id)
 
-    def get_jobs_by_run_id(self, run_id: str) -> List[Job]:
-        return self.value[0].get_jobs_by_run_id(run_id)
+    def run_is_inline(self, run_id: str) -> bool:
+        # looking for jobs to determine inline is only valid
+        # since we know the run has at least reached SCHEDULED due to it
+        # not being CREATED.
+        run_is_inline = True
+        if self is ObjectSource.DB:
+            has_new_jobs = db_queries.count_jobs_by_run_id(run_id) > 0
+            if has_new_jobs:
+                run_is_inline = False
```

Comment:
I believe you can simplify the logic and increase readability by just returning in spots where you update the `run_is_inline` variable.

Reply:
yes, it was only there to enable the debug statement which I have now removed